l10n_it_fatturapa_out/data/invoice_it_template.xml

```diff
@@ -192,12 +192,11 @@ e 'line' per riga di fattura (a seconda del livello in cui sono chiamati)
                     t-if="line.admin_ref"
                     t-esc="line.admin_ref"
                 />
-                <!--                <AltriDatiGestionali>-->
-                <!--                    <TipoDato t-esc=""/>-->
-                <!--                    <RiferimentoTesto t-esc=""/>-->
-                <!--                    <RiferimentoNumero t-esc=""/>-->
-                <!--                    <RiferimentoData t-esc=""/>-->
-                <!--                </AltriDatiGestionali>-->
+                <AltriDatiGestionali
+                    t-if="tax_ids.kind_id.code and tax_ids.kind_id.code == 'N2.1' and line.move_id.partner_id.country_id.code in ['AT', 'BE', 'BG', 'CY', 'HR', 'DK', 'EE', 'FI', 'FR', 'DE', 'GR', 'IE', 'IT', 'LV', 'LT', 'LU', 'MT', 'NL', 'PL', 'PT', 'CZ', 'RO', 'SK', 'SI', 'ES', 'SE', 'HU']"
```

Comment:
Potresti estrarre questa condizione in un metodo ed eseguirla lato backend?
Potrebbe essere utile anche in altri punti poi non è il massimo mettere troppo codice sotto forma di stringa perché non viene controllato dai linters ed è complicato capirne gli errori se ce ne dovessero essere.

La lista delle nazioni sai mica da dove viene? Non l'ho trovata nelle specifiche riportate nella issue.
cc @sergiocorato

Reply:
@SimoRubi sono le nazioni UE

Reply:
ah ecco, effettivamente ci potevo arrivare :) quindi si potrebbe sfruttare il gruppo di nazioni "Europa"?
https://github.com/odoo/odoo/blob/7f895cf2d69ec8fa07e8736c637aef50f5a8506d/odoo/addons/base/data/res_country_data.xml#L1592-L1601

Reply:
Sì, io mi sono limitato a riportare le modifiche di #2096 il più fedelmente possibile col senno di poi potevo fare un lavoro più intelligente, vd. come è adesso.

Reply:
Se è affidabile sì, nelle versioni precedenti non era aggiornato, però ci esponiamo al rischio che l'utente intervenga togliendo o mettendo una nazione errata.